packages/flame/lib/src/game/flame_game.dart

```diff
@@ -150,4 +154,13 @@ class FlameGame extends Component with Game {
 
   @override
   Projector get projector => camera.combinedProjector;
+
+  ComponentNotifier<T> componentNotifier<T extends Component>() {
```

Comment:
The naming of this sounds a bit like it would be listening to a single component since the word is in singular, maybe name it `componentsNotifier`?